sdk/src/main/java/ch/admin/bag/covidcertificate/eval/repository/TrustListRepository.kt

```diff
@@ -63,7 +64,7 @@ internal class TrustListRepository(
 		}
 	}
 
-	private suspend fun refreshCertificateSignatures(forceRefresh: Boolean) = withContext(Dispatchers.IO) {
+	private suspend fun refreshCertificateSignatures(forceRefresh: Boolean): Unit = withContext(Dispatchers.IO) {
```

Comment:
Add flag to indicate if this was called recursively to prevent an endless loop if the backend is faulty